docs/follower_node.md

```diff
@@ -0,0 +1,56 @@
+# Follower Node
+
+When started with `"EnableFollowMode": true`, algod starts with a special
+property which allows it to be paused. This allows software to be written
+which runs synchronously with a node. If you need account balances at each
+round, or at a particular round, this is the only way to do it. On start
+the node will be paused, the REST API is used to resume and select a new
+round.
+
+It also allows fetching a `Ledger State Delta` object for recent rounds.
+This object is not designed for external users, but may be useful for
+advanced applications if you're willing to figure things out. It contains
+complete information for transitioning a database to the next round,
+including new account balances, changes to application and asset states,
+and new box information. Such information was previously unavailable to
+application developers.
+
```

Comment:
I would hint at the Restrictions section below.... e.g. "Note that a follower node is incompatible with certain other node functionalities, see Restrictions."